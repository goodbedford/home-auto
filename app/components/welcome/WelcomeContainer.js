import React from "react";
import {Route, Link, Redirect} from "react-router-dom";
import Welcome from "./Welcome";
import httpHelper from "../../utils/httpHelper.js";

const propTypes =  {
};
const defaultProps = {
  msg: "Premium Comfort"
};

class WelcomeContainer extends React.Component {
  constructor(props) {
    super(props);
    this.state = {
    };
  }
  componentDidMount () {
  }

  render() {
    return (
      <Welcome
        msg={this.props.msg}
      />
    );
  }
}
WelcomeContainer.propTypes = propTypes;
WelcomeContainer.defaultProps = defaultProps;

export default WelcomeContainer;
