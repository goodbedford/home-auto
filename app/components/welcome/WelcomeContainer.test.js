/* eslint-disable no-undef */

import React from "react";
import {shallow} from "enzyme";
import toJson from "enzyme-to-json";
import WelcomeContainer from "./WelcomeContainer";

describe("<WelcomeContainer />", () => {
  const props =  {msg: "welcome"};
  it("WelcomeContainer snapshot", () => {
    const wrapper = shallow(<WelcomeContainer {...props} />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
