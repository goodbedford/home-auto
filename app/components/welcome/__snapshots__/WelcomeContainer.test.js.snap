// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WelcomeContainer /> WelcomeContainer snapshot 1`] = `
<Welcome
  msg="welcome"
/>
`;
